packages/sdk-snippets/CONTRIBUTING.md

```diff
@@ -0,0 +1,110 @@
+# Contributing
+
+For general contributing and commit guidelines for this project check out https://github.com/readmeio/.github/blob/main/.github/CONTRIBUTING.md.
+
+## Creating a new Target
+
+Much like in [HTTPSnippet](https://npm.im/httpsnippet) targets are programming languages. If you wish to create a new snippet generator for a new language follow these steps:
+
+1. Create a new directory in `src/targets/` and name it the language you're targeting. (eg. for PHP it'd be `php`, .NET `dotnet`).
+2. Create a `target.ts` file in this new directory with code similar to the following:
+
+```ts
+import type { Target } from '../targets';
+import { express } from './express/webhooks/client';
+
+export const node: Target = {
+  info: {
+    // This should be identical to the directory you created.
+    key: 'node',
+
+    // The proper naming of this language.
+    title: 'Node.js',
+
+    // The file extension that this language prefers. If it doesn't have a file
+    //  extension you can set this to an empty string.
+    extname: '.js',
+
+    // The default client this language will use. See below for instructions on
+    // creating a client.
+    default: 'native',
+
+    // This is how you execute the code that it generates.
+    cli: 'node %s',
+  },
+  services: {
+    server: {
+      clientsById: {},
+    },
+    webhooks: {
+      clientsById: {
+        express, // The Client in question
+      },
+    },
+  },
+};
+```
+
+3. Once you have a target created you need to let the library know it exists. You can do this by loading and exporting it within the `targets` const in `src/targets.ts`. See that file for some available examples.
+
+## Creating a new Client
```

Comment:
Nit: Consider using "framework" over "client"

Reply:
`target` and `client` are pretty ingrained into HTTPSnippet so I'm hesitant to change the language.